crates/spfs/src/proto/defs/database.proto

```diff
@@ -39,7 +39,8 @@ message FindDigestsRequest { DigestSearchCriteria search_criteria = 1; }
 message FindDigestsResponse {
     oneof result {
         Error error = 1;
-        Digest ok = 2;
+        // 2 was used
```

Comment:
I think reserving the id is the recommended way to do this

https://protobuf.dev/programming-guides/proto3/#fieldreserved